network/networkutils/networkutils_linux.go

```diff
@@ -176,6 +176,12 @@ func AllowIPAddresses(bridgeName string, skipAddresses []string, action string)
 	return nil
 }
 
+func BlockWireserverTraffic() error {
+	// iptables -t filter -I FORWARD -j DROP -d <wireserver ip>/32 -p tcp -m tcp --dport 80
+	dropWireserver := fmt.Sprintf("-d %s/32 -p tcp -m tcp --dport 80", AzureDNS)
```

Comment:
should drop for udp as well

Reply:
Replicating capz rule to block for tcp protocol only as we discussed.